pkg/vm/engine/tae/containers/utils.go

```diff
@@ -40,24 +41,19 @@ func ApplyUpdates(vec Vector, mask *roaring.Bitmap, vals map[uint32]any) {
 
 func FillBufferWithBytes(bs *Bytes, buffer *bytes.Buffer) *Bytes {
 	buffer.Reset()
-	offBuf := bs.OffsetBuf()
-	lenBuf := bs.LengthBuf()
-	dataBuf := bs.Data
-	size := len(offBuf) + len(lenBuf) + len(dataBuf)
+	size := bs.Size()
 	if buffer.Cap() < size {
 		buffer.Grow(size)
 	}
 	nbs := NewBytes()
+	nbs.FixedTypeSize = bs.FixedTypeSize
```

Comment:
What is the diff between FixedTypeSize and FixedType?

Reply:
Removed